codepropertygraph/src/main/scala/io/shiftleft/passes/CpgPass.scala

```diff
@@ -26,6 +26,7 @@ import scala.concurrent.duration.DurationLong
   * @param cpg
   *   the source CPG this pass traverses
   */
+@deprecated
```

Comment:
can you please add the `description` and `since` flags here, so that users understand what's going on and know about alternatives?